src/encoded/static/components/gene_search/input.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+
+/**
+* Controlled input box that accepts user's query
+* and triggers onChange callback.
+*/
+const Input = React.forwardRef((props, ref) => (
```

Comment:
It also seemed to work when I passed the `ref` in `props` under a different name, but was concerned something would break if I didn't use `forwardRef`.